internal/stats/metrics_recorder_list.go

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2024 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package stats
+
+import (
+	estats "google.golang.org/grpc/experimental/stats"
+	"google.golang.org/grpc/grpclog"
+	"google.golang.org/grpc/stats"
+)
+
+var logger = grpclog.Component("metrics-recorder-list")
+
+// MetricsRecorderList forwards Record calls to all of it's metricsRecorders.
```

Comment:
its*.  Check throughout please.

Reply:
Done. I control fed and it seems like this is the only instance.

Reply:
I would like to propose that you never write "it's" and use either "its" or "it is" from now on.  No more apostrophes for you!  ;)